attending/downloader.py

```diff
@@ -0,0 +1,24 @@
+import urllib.request as request
```

Comment:
I'm not super excited that this file is called `downloader.py`. I think we can do better. I'll have to think about it.